cmd/kops/delete.go

```diff
@@ -42,24 +42,30 @@ type DeleteOptions struct {
 
 var (
 	delete_long = templates.LongDesc(i18n.T(`
-	Delete clusters, instancegroups, or secrets.
+	Delete Kubernetes clusters, instancegroups, and or secrets.
```

Comment:
From my grammar checker "The use of and/or is severely frowned upon in formal writing. Consider using only one conjunction or rewriting the sentence."